english/turing-test/ring/one_ring.md

```diff
@@ -8,3 +8,5 @@ Meanwhile, you descry your surroundings for escape routes.
 You eventually realize that tickling the walls makes the whole room to rumble and cracks start appearing in the walls.
 
 The cracks in the walls grow larger and larger until they form a doorway big enough for you to walk thorugh.
+
+A shrill sound almost like someone scraping their fingernails on a chalkboard emanates from the doorway, growing louder with each passing moment.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.